app/controllers/application_controller.rb

```diff
@@ -7,7 +7,13 @@ class ApplicationController < ActionController::Base
   before_filter :authorize!
   before_filter :fedora_setup
 
-  rescue_from ActiveFedora::ObjectNotFoundError, with: -> { render text: 'Object Not Found', status: :not_found }
+  rescue_from ActiveFedora::ObjectNotFoundError do
+    render text: "Object Not Found", status: :not_found
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.